homeassistant/components/light/mqtt.py

```diff
@@ -192,6 +192,10 @@ def turn_on(self, **kwargs):
         """Turn the device on."""
         should_update = False
 
+        if not self._state:
```

Comment:
Do you think it would be better creating a new light component based on the mqtt light component that would be specific for the easyesp platform? would that make integration into main branch easier? Lets say a eemqtt.py component? that way it would not compromise other use case.